Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -251,7 +251,12 @@ class SignalAttachment: NSObject {
     // Image attachments may be converted to another image format before 
     // being uploaded.
     private class var inputImageUTISet: Set<String> {
-        return MIMETypeUtil.supportedImageUTITypes().union(animatedImageUTISet)
+         // HEIC is valid input, but not valid output. Non-iOS11 clients do not support it.
+        let heicSet: Set<String> = Set(["public.heic"])
+
+        return MIMETypeUtil.supportedImageUTITypes()
```

Comment:
My understanding is that animated .heifs/heics will be used for "live images".  I'm wondering if we should support sharing those (as .gifs?)?

Reply:
That might be a good idea. I've made a card.